packages/fluent-theme/src/components/sendBox/Toolbar.module.css

```diff
@@ -14,9 +17,9 @@
   color: currentColor;
   cursor: pointer;
   display: flex;
+  height: var(--webchat__sendbox-button-height);
   justify-content: center;
   padding: 3px;
```

Comment:
Is this value accurate

Reply:
It was `width: 32px` and `aspect-ratio: 1`.

I think `height: 32px` and `aspect-ratio: 1` is better because, just in case of a rectangular icon, we don't make the send box taller.

Reply:
Not to mention I want to keep the send box height the same when switching between `blueprint` and normal.

Maybe I should write a test for it too. 🤔